README.md

```diff
@@ -21,6 +25,44 @@ You may have noticed that specifying "production" created an artifact from the m
 
 Before the artifact is built Jarå will check that _branch_name_ and origin/*branch_name* point to the same commit. The reason for this is so that you don't release an artifact with a SHA that is not visible to others (this does not check that you've pulled before you release, but the important thing is to not release something that is not trackable).
 
+The same can be accomplished by running this from the command line:
+
+```
+$ jara --environment production --release --bucket artifact-bucket
+```
```

Comment:
I think `--release` should not be specified in the middle of the command in the example. Perhaps it would make sense to treat as a subcommand rather than a plain option:
```
$ jara release --environment production --bucket artifact-bucket`
```